src/audio/module_adapter/library/native_system_service.c

```diff
@@ -123,10 +121,19 @@ AdspErrorCode SystemServiceSendNotificationMessage(NotificationTarget notificati
 	return ADSP_NO_ERROR;
 }
 
-AdspErrorCode SystemServiceGetInterface(AdspIfaceId id, SystemServiceIface  **iface)
+AdspErrorCode native_system_service_get_interface(adsp_iface_id id, system_service_iface  **iface)
 {
 	if (id < 0)
 		return ADSP_INVALID_PARAMETERS;
 	return ADSP_NO_ERROR;
 }
 
+struct native_system_service_api native_system_service = {
+	.log_message  = native_system_service_log_message,
+	.safe_memcpy = native_system_service_safe_memcpy,
+	.safe_memmove = native_system_service_safe_memmove,
+	.vec_memset = native_system_service_vec_memset,
+	.notification_create = native_system_service_create_notification,
+	.notification_send = native_system_service_send_notif_msg,
+	.get_interface = native_system_service_get_interface
```

Comment:
Long term we can use Zephyr runtime linking when it's ready so modules can be more compatible with base FW Kconfig changes.